sql/table.h

```diff
@@ -800,6 +800,18 @@ struct TABLE_SHARE
       return &period;
     return NULL;
   }
+
+  Field *vers_start_field()
```

Comment:
Now it duplicates `period_info_t`'s interface. Versioning has specifics, but not in this place. In future, we could unify the `SYSTEM_TIME` and `apllication_time` periods access to user, and even look forward into multi-period tables -- that's why I wanted one common interface for them.

Reply:
Well, maybe we can pass it through now, until we introduce some field-like access for periods, or even make them virtual fields of type `PERIOD`, maybe

Reply:
Just for patch cleanliness I decided to leave it for now. The subject for later refactoring.